app/views/layouts/ats/application.html.slim

```diff
@@ -16,7 +16,6 @@ html
     link href="/icon#{'-dev' if Rails.env.development?}.svg" rel="icon" type="image/svg+xml"
     link href="/apple-touch-icon.png" rel="apple-touch-icon"
     link href="/site.webmanifest" rel="manifest"
-    link color="#d52b1e" href="/safari-pinned-tab.svg" rel="mask-icon"
```

Comment:
According to [this](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs), no need for safari icon. Also, all online converters removed this icon from the generated results